### ISSUE TITLE
netbox_device_bay_template module isn't found

### ISSUE THREAD

**Issue by adamwoolhether** [netbox-community/ansible_modules#268]
##### ISSUE TYPE

 - Bug Report

###### Ansible:

ansible 2.9.10
config file = /etc/ansible/ansible.cfg
configured module search path = ['/root/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
ansible python module location = /usr/lib/python3.6/site-packages/ansible
executable location = /usr/bin/ansible
python version = 3.6.8 (default, Apr 16 2020, 01:36:27) [GCC 8.3.1 20191121 (Red Hat 8.3.1-5)]
###### Netbox:

v2.8.6
pynetbox (4.3.3)
###### Collection:

v0.2.3
##### SUMMARY

netbox_device_bay_template is not found. I can use netbox_device_bay just fine.

##### STEPS TO REPRODUCE

```yaml
---
- name: "PLAY: SETUP MULTI NODE BAYS"
  connection: local
  hosts: localhost
  collections:
    - netbox.netbox

  tasks:
    - name: "TASK 1: SETUP MULTI NODE BAYS"
      netbox_device_bay_template:
        netbox_url: https://1XXX
        netbox_token: XXX
        validate_certs: no
        state: present
        data:
          name: "{{ item.name }}"
          device_type: "{{ item.dev }}"
          description: Node
      loop:
........
```

##### EXPECTED RESULTS

A new bay template for the targeted device type
##### ACTUAL RESULTS

ERROR: couldn't resolve module/action.

```
ERROR! couldn't resolve module/action 'netbox_device_bay_template'. This often indicates a misspelling, missing collection, or incorrect module path.

The error appears to be in '/ansible/provision/test_bay_temp.yml': line 9, column 7, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

  tasks:
    - name: "TASK 1: SETUP MULTI NODE BAYS"
      ^ here
This one looks easy to fix. It seems that there is a value started
with a quote, and the YAML parser is expecting to see the line ended
with the same kind of quote. For instance:

    when: "ok" in result.stdout

Could be written as:

   when: '"ok" in result.stdout'

Or equivalently:

   when: "'ok' in result.stdout"
```

**Comment by toerb** [comment]
`netbox_device_bay_template` is not present in v0.2.3:
https://github.com/netbox-community/ansible_modules/tree/v0.2.3/plugins/modules

It was added 4 days ago in #251 and will be included in v0.2.4.

**Comment by adamwoolhether** [comment]
didn't realize that. thank you!!
